docs/pages/guides/building-for-tv.mdx

```diff
@@ -0,0 +1,224 @@
+
+<Terminal cmd={['npx expo install expo@next', 'npx expo install --fix']} cmdCopy="npx expo install expo@next && npx expo install --fix" />
+
+## Limitations
+
+Some Expo features are not supported at this time:
+
+- Dev client and dev launcher
+- Expo Router
+
+<Step label="1">
+
+## Modify dependencies for TV
+
+In **package.json**, modify the `react-native` dependency to use the TV repo, and exclude this dependency from expo install changes.
+
+{/* prettier-ignore */}
+```json package.json
+{
+  /* @hide ... your existing package.json */ /* @end */
+  "dependencies": {
+    /* @hide ... your existing dependencies */ /* @end */
+    "react-native": "npm:react-native-tvos",
+    /* @hide ... your existing dependencies */ /* @end */
+  },
+  "expo": {
+    "install": {
+      "exclude": [
+        "react-native"
+      ]
+    }
+  }
+}
+```
+</Step>
+
+<Step label="2">
+
+## Add the TV config plugin
+
+<Terminal cmd={['npx expo install @react-native-tvos/config-tv']} />
+
+Installing the package adds the `@react-native-tvos/config-tv` plugin to your project's **app.json**.
+
+When installed, the plugin will modify the project for TV when either
+
+- The environment variable `EXPO_TV` is set to "1"
+- The plugin parameter `isTV` is set to true
+
+Optionally set `showVerboseWarnings` to true to see additional information during prebuild.
+
+```json app.json
+{
+  "plugins": [
+    [
+      "@react-native-tvos/config-tv",
+      {
+        "showVerboseWarnings": true
+      }
+    ]
+  ]
+}
+```
+
+</Step>
+
+<Step label="3">
+
+## Run prebuild
+
+Set the `EXPO_TV` environment variable, and run prebuild to make the TV modifications to the project.
+
+<Terminal cmd={['export EXPO_TV=1', 'npx expo prebuild --clean']} />
+
+> **Note**: The `--clean` argument is recommended, and is required if you have existing Android and iOS directories in the project.
+
+</Step>
+
+<Step label="4">
+
+## Build for Apple TV
+
+After running prebuild, run the following command to build and run the app on an Apple TV simulator:
+
+<Terminal
+  cmd={[
+    'npx expo run:ios'
+  ]}
+/>
+
+</Step>
+
+<Step label="5">
+
+## Build for Android TV
+
+After running prebuild, start an Android TV emulator, and then use the same command as you would to start the app on an Android phone emulator.
+
+<Terminal
+  cmd={[
+    'npx expo run:android'
+  ]}
+/>
+
+</Step>
+
+<Step label="6">
+
+## Revert TV changes and build for phone
+
+You can revert the changes for TV and go back to phone development by unsetting `EXPO_TV` and running prebuild again:
+
+<Terminal cmd={['unset EXPO_TV', 'npx expo prebuild --clean']} />
+
+</Step>
+
+<Step label="7">
+
+## Create EAS build profiles for both TV and phone
+
+Since the TV build is driven by the value of an environment variable, it is easy to set up EAS build profiles that build from the same source but target TV instead of phone.
```

Comment:
```suggestion
Since the TV build is driven by the value of an environment variable, it is easy to set up EAS Build profiles that build from the same source but target TV instead of phone.
```